src/main/java/ch/njol/skript/effects/EffRepeat.java

```diff
@@ -0,0 +1,80 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.effects;
+
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.util.Kleenean;
+
+@Name("Repeat Effect")
+@Description("Runs an effect x number of times.")
+@Examples({"send \"<red>Server will restart in 1 minute!\" to all players 3 times! "})
+@Since("INSERT VERSION")
+public class EffRepeat extends Effect  {
+
+	static {
+		Skript.registerEffect(EffRepeat.class, "(repeat|run) [effect] <.+> %number% times");
```

Comment:
In my opinion `(repeat|run)` should be optional. I dont feel it makes sense to type:
`run send "hi" to player 10 times`

It would make more sense just to do:
`send "hi" to player 10 times`

Reply:
I agree. I just wasn't sure if that would cause any conflicts.